src/main/java/com/example/dto/user/UserRegistrationRequestDto.java

```diff
@@ -0,0 +1,28 @@
+package com.example.dto.user;
+
+import com.example.validation.Email;
+import com.example.validation.FieldMatch;
+import com.example.validation.Password;
+import jakarta.validation.constraints.NotBlank;
+import lombok.Data;
+
+@FieldMatch.List({
```

Comment:
```suggestion
@FieldMatch(
        fields = {"password", "repeatedPassword"},
        message = "Password and repeated password must be equal"
)
```

Reply:
fields = {"password", "repeatedPassword"} - doesn't work for me, replaced with:
 
@FieldMatch(
    first = "password", 
    second = "repeatedPassword",
    message = "Password and repeated password must be equal"
)